docs/conf.py

```diff
@@ -110,10 +110,28 @@ def wiki_role(role, rawtext, text, lineno, inliner, options={}, content=[]):
     node = nodes.reference(rawtext, title, refuri=ref, **options)
     return [node], []
 
+packet_matcher = re.compile(r"(.*) <(.*)>")
```

Comment:
I thought there was built-in functionality for this, but I'm not actually sure where to find that right now. That's not a blocking concern though if it works.

Reply:
There's one for numbers but this one is for all text - if needed I could just use this one in place of the other?

Reply:
So I found the code that takes care of this, but I am not sure how to invoke it.  I spent a couple hours trying some variations of Add_role_to_domain but none of them implemented the role as wanted here.  Would it be more preferred to figure out how the built in version works?

Reply:
Still unsure how to do this in Sphinx.  Left a comment about it, but I dont think I should spend more time on it on my own since my time box for it has expired (I spent about 4 hours on it).  If you know how to get it to work please let me know and i'll work on it!